nixos/modules/virtualisation/containers.nix

```diff
@@ -134,6 +134,9 @@ let
         --setenv HOST_ADDRESS6="$HOST_ADDRESS6" \
         --setenv LOCAL_ADDRESS6="$LOCAL_ADDRESS6" \
         --setenv PATH="$PATH" \
+        ${if cfg . "additionalCapabilities" or null != null then
+          ''--capability="${concatStringsSep " " cfg . "additionalCapabilities"}"'' else ""
```

Comment:
I mostly stuck to the patterns that were already present in `containers.nix`. Thanks for fixing it, will update/rebase.